src/main/java/br/com/casadocodigo/loja/AppWebConfiguration.java

```diff
@@ -0,0 +1,151 @@
+package br.com.casadocodigo.loja;
```

Comment:
http://docs.spring.io/spring-boot/docs/current/reference/htmlsingle/#using-boot-locating-the-main-class